data/modules/general/threat_modeling.yml

```diff
@@ -0,0 +1,94 @@
+category: General
+description: Have you performed a theat assessment on your application (aka a Threat Model)?
+minimumRisk: Low Risk
+checkLists:
+  Diagram:
+      - question: We have created a diagram of the software, can tell a story about the system and consider what could go wrong without using words such as "sometimes" or "also".
+      - question: The diagram show all the [trust boundaries](https://en.wikipedia.org/wiki/Trust_boundary) and cover all application roles, and all network interfaces.
```

Comment:
rather than modelling the threat modelling process, could this map the results of a threat model?

E.g. 
```
 - question: We've have created a baseline threat model
 - question: We've updated our baseline threat model with this new feature
```

Depends on how much STRIDE (or similar) knowledge we want teams to have? vs ensuring they've performed a threat model, and are tracking mitigations/risks accordingly

Reply:
This is a good point and i might end up removing the stride threats in the checklist and just reference them instead. But think we should keep it for now. Also id like to add a better description field so that we can add some steps to follow in the description field in Trello. But will create a new PR for that feature :)